client/src/game/ui/menu/menu.vue

```diff
@@ -68,6 +68,21 @@
                                 <div v-if="!notes.length">No notes</div>
                             </div>
                         </div>
+                        <!-- MARKERS -->
```

Comment:
I would move the markers section out of the DM only section, I can see some players also wanting to jump around between markers in some circumstances.

Reply:
Done.